_1327/templates/base.html

```diff
@@ -95,6 +95,18 @@
 							</ul>
 						</li>
 					{% endif %}
+					{# Search form #}
+					<li class ="pull-right">
+						<a href="#" id="popover" data-placement="bottom">
+							<span class="glyphicon glyphicon-search" aria-hidden="true"></span>
+						</a>
+							<div id="popover-content" class="hide">
+							  	<form action='/search' method='post' id='general_search'>
```

Comment:
could you use the `{% url 'main:search' %}` template tag, here? Should look like this, I'm not 100% sure.